pyccl/halos/massdef.py

```diff
@@ -0,0 +1,247 @@
+from .. import ccllib as lib
+from ..core import check
+from ..background import species_types, rho_x, omega_x
+import numpy as np
+
+
+def mass2radius_lagrangian(cosmo, M):
+    """ Returns Lagrangian radius for a halo of mass M.
+    The lagrangian radius is defined as that enclosing
+    the mass of the halo assuming a homogeneous Universe.
+
+    Args:
+        cosmo (:obj:`Cosmology`): A Cosmology object.
+        M (float or array_like): halo mass in units of M_sun.
+
+    Returns:
+        float or array_like: lagrangian radius in Mpc
+    """
+    return (M / (4.18879020479 * rho_x(cosmo, 1, 'matter')))**(1./3.)
+
+
+def convert_concentration_py(cosmo, c_old, Delta_old, Delta_new):
```

Comment:
Why is this routines name subscripted with `py`?

Reply:
No good reason. I thought there was a `convert_concentration` swig function, but there isn't. Changed!